Source/WebCore/dom/CustomElementRegistry.cpp

```diff
@@ -128,6 +128,17 @@ JSC::JSValue CustomElementRegistry::get(const AtomString& name)
     return JSC::jsUndefined();
 }
 
+JSC::JSValue CustomElementRegistry::getName(JSC::JSGlobalObject& globalObject, JSC::JSValue constructorValue)
+{
+    JSC::JSObject* constructor = constructorValue.getObject();
+    if (!constructor)
+        return JSC::jsUndefined();
+    auto* interface = findInterface(constructor);
```

Comment:
Can't use the name "interface" on Windows. It's why WinCairo is failing to build.
This comes up every time I touch this file. It's annoying as hell...